Marlin/src/module/settings.cpp

```diff
@@ -2469,7 +2469,7 @@ void MarlinSettings::reset() {
   //
 
   #if HAS_FILAMENT_SENSOR
-    runout.enabled = true;
+    runout.enabled = false;
```

Comment:
please do not disable it.... by default

Reply:
I'm curious about the logic on why? There are many board that ship with a filament sensor port - and the majority never get connected to a filament sensor and remain unpopulated. Why would it not be sane to have a default of off - and not require every user with a board that features a filament sensor which remains unused to turn the feature off or they won't be able to print?

This leaves the upgrade path for an end user to be able to upgrade to using a filament sensor by plugging it in and turning it on.

Personal preferences aside, I can't see the reasoning behind leaving it as a 'broken by default' behaviour for printing.

Reply:
In this case, maybe a new config define to set it on by default is required... Most "ready to use"printers have it these days... sure DIY ones are different (standalone motherboards)

Reply:
Most “ready to use printers” are not hacked by industry users (Ultimaker, Lulzbot, and Prusa to an extent). I have used all 3 professionally and at that level you don’t hack them, you use them for the intended purpose and buy a new printer if you need other capabilities. 

You’re argument for not changing it is to take an open source firmware and tailor it to private industry. Who have their own engineering teams to build, test, and debug their releases. If you start talking about ready to use printers in terms of Creality, Anet, Reprap, ect. Then I would say I have yet to see someone who bought one and kept it stock. But at that point, without solid data, it is impossible to say what the huge community actually does or dosn’t do. But why not leave it up to them?

Also. none of those printers in the latter group come with filament runout sensors as of yet. So an argument to tailor to hackable printers and the community is appropriate, but in that case it would be default off. 

For those wise enough to put a sensor on, they would be aware of the need to support it. Hiding a setting from users who would not be looking for it is as one person has said on a facebook group “is a hostile default”.

Reply:
I concur. It is not my preference either.